src/pages/view/TitleBar/ActionsBar.js

```diff
@@ -226,6 +234,15 @@ const ViewActions = ({
         </DropdownButton>
     )
 
+    const updateStarredFilter = () => {
+        const isStarred = isFilterStarred(starredFilter)
+        const value = isStarred ? '' : STARRED_STATE
```

Comment:
The other similar endpoint is:

/api/userDataStore/dashboard/showDescription

which returns a boolean instead of a string. Can we do something similar? (unless we think we will have more states for the flag)